### PR TITLE
Connectome work bench

### PR DESCRIPTION
**Pull request by rtripath89**

### CODE REVIEW

File: easybuild/easyconfigs/c/ConnectomeWorkbench/ConnectomeWorkbench-1.2.2.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'ConnectomeWorkbench'
+version = '1.2.2'
+
+homepage = 'http://www.humanconnectome.org/software/get-connectome-workbench.html'
```
Comment (boegel) [inline easybuild/easyconfigs/c/ConnectomeWorkbench/ConnectomeWorkbench-1.2.2.eb]: http://www.humanconnectome.org/software/connectome-workbench.html looks better as homepage?

File: easybuild/easyconfigs/c/ConnectomeWorkbench/ConnectomeWorkbench-1.2.2.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+program for performing a variety of algorithmic tasks using volume, 
+surface, and grayordinate data."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://ftp.humanconnectome.org/workbench/']
+sources = ['workbench-rh_linux64-v%(version)s.zip']
+
+modextrapaths = {'PATH': 'bin_rh_linux64', 'LIBRARY_PATH': 'libs_rh_linux64', 
+'LD_LIBRARY_PATH': 'libs_rh_linux64'}
```
Comment (boegel) [inline easybuild/easyconfigs/c/ConnectomeWorkbench/ConnectomeWorkbench-1.2.2.eb]: please fix formatting:

```python
modextrapaths = {
    'LD_LIBRARY_PATH': 'libs_rh_linux64',
    'LIBRARY_PATH': 'libs_rh_linux64',
    'PATH': 'bin_rh_linux64',
}
```

**Comment by rtripath89** [comment]
@boegel Updated the suggested changes.

File: easybuild/easyconfigs/c/ConnectomeWorkbench/ConnectomeWorkbench-1.2.2.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://ftp.humanconnectome.org/workbench/']
+sources = ['workbench-rh_linux64-v%(version)s.zip']
+
+modextrapaths = {
+     'PATH': 'bin_rh_linux64', 
+     'LIBRARY_PATH': 'libs_rh_linux64', 
+     'LD_LIBRARY_PATH': 'libs_rh_linux64'}
```
Comment (boegel) [inline easybuild/easyconfigs/c/ConnectomeWorkbench/ConnectomeWorkbench-1.2.2.eb]: @rtripath89 please move the `}` to the next line:

```python
modextrapaths = {
    ...
    'PATH': 'bin_rh_linux64',
}
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/27844d8820ba3d029e3dcabe1ace7ac5 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2105.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/36ff0bfc496db2ec02e3a060bfb48652 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @rtripath89!

**Pull request merged**